app/Http/Controllers/CompetitionsController.php

```diff
@@ -16,7 +25,8 @@ class CompetitionsController extends Controller
      */
     public function index()
     {
-        return view('competitions.index');
+        $competitions = Competition::all();
+        return view('competitions.index')->withCompetitions($competitions);
```

Comment:
Sorryz didn't realize it was a problem, I was just referencing Toto since I'm still new to this https://github.com/DoSomething/toto/pull/5/files#diff-0b13542aaa91c162d7dec98854338206R25